readme.txt

```diff
@@ -1,8 +1,8 @@
 === WooPayments - Fully Integrated Solution Built and Supported by Woo ===
 Contributors: woocommerce, automattic
 Tags: payment gateway, payment, apple pay, credit card, google pay, woocommerce payments
-Requires at least: 6.1
-Tested up to: 6.3
+Requires at least: 6.0
+Tested up to: 6.2
```

Comment:
I'm tempted to leave this as 6.3 but goal of this PR is to revert https://github.com/Automattic/woocommerce-payments/pull/7050, so I went ahead to revert this line as well.